lib/plugins/aws/deploy/lib/updateStack.js

```diff
@@ -31,10 +29,14 @@ module.exports = {
         'CAPABILITY_IAM',
       ],
       Parameters: [],
-      TemplateBody: JSON.stringify(this.serverless.service.resources.aws),
+      TemplateBody: JSON.stringify(this.serverless.service.resources),
     };
 
-    return this.CloudFormation.updateStackPromised(params);
+    return this.sdk.request('CloudFormation',
+      'updateStack',
+      params,
+      this.options.stage,
+      this.options.region);
```

Comment:
Do these options overwrite the defaults defined in the service / `serverless.yaml` file?